Controllers/LoneAntController.cs

```diff
@@ -12,88 +12,87 @@
 
 namespace CodeChallengeInc.SubmissionApi.Controllers
 {
-	[Route("lone-ant")]
 	[ApiController]
-	public class LoneAntController : ControllerBase, IGameController
+	public class GameSubmissionController : ControllerBase, IGameController
 	{
 		internal string Passcode { get; set; }
 		IFileService _fileService;
-		public LoneAntController(IFileService fileService)
+		public GameSubmissionController(IFileService fileService)
 		{
 			_fileService = fileService;
 			Passcode = (Environment.GetEnvironmentVariable(EnvironmentConstants.PasscodeVariable) != null) ? Environment.GetEnvironmentVariable(EnvironmentConstants.PasscodeVariable) : EnvironmentConstants.DefaultPasscode;
```

Comment:
This never really panned out but this passcode would be used for anything we wanted to limit to admins.